File: plugin/js/db/cassandra/v1_x/methods.js
function detect(input, callback) {
  let path = input.request.root;
  callback(null, { value: true, intValue: null, error: null });
  // callback(reject, { value: null, error: reject })
}

function isDbUsed(input, callback) {
  let path = input.request.root;
  callback(null, { value: true, error: null });
  // callback(reject, { value: null, error: reject })
}

function percentOfDbUsed(input, callback) {
  let path = input.request.root;
  callback(null, {  error: null });
  // callback(reject, { value: null, error: reject })
}

module.exports = {
  detect: detect,
  isDbUsed: isDbUsed,
  percentOfDbUsed: percentOfDbUsed,
};

File: plugin/python/db/postgres/v1_x/methods.py
from protos.plugin import db_pb2_grpc, common_pb2, db_pb2


class Methods(db_pb2_grpc.DbServiceServicer):
    def Detect(self, request, context):
        return db_pb2.ServiceOutputBoolInt(value=True, error=None)

    def IsDbUsed(self, request, context):
        return common_pb2.ServiceOutputBool(value=True, error=None)

    def PercentOfDbUsed(self, request, context):
        return common_pb2.ServiceOutputFloat(value=55.6, error=None)
